docker-compose/cassandra-4.0/README.md

```diff
@@ -71,7 +71,7 @@ Both convenience scripts support the following options:
 
 * When using the convenience scripts, the Stargate Docker image (`SGTAG`) used for the `-l` option is the current (snapshot) version as defined in the top level project `pom.xml` file. It can be overridden with the `-t` option on either script:
 
-  `./start_cass_40.sh -t v2.0.0-BETA-2`
+  `./start_cass_40.sh -t v2.0.0`
```

Comment:
This should be `v2.1.0`?

Reply:
Since it is just an example provided for documentation I don't think it matters what version is used.